data/src/split.py

```diff
@@ -60,10 +29,13 @@ def _iterate_through_file(
 
 def main() -> None:
     tsv_path = sys.argv[1]
```

Comment:
I realize this is not new but we really ought to be using argparse for this, just for consistency. If called without an argument it'll crash, but with an uninformative argument.